### PR TITLE
Bump minimum VS Code version to 1.57 in ui package

### PR DESCRIPTION
**Pull request by ejizba**
The primary reason I want to do this is because VS Code changed to Node 14 in v1.56 (see [here](https://code.visualstudio.com/updates/v1_56#_electron-12-update)) and I prefer to keep our Node version in line with them.

However, if I bump just one more version to 1.57 (released on June 10), then I can use VS Code's built-in loading icon since https://github.com/microsoft/vscode/issues/117859 was fixed.

### CODE REVIEW

File: ui/src/utils/randomUtils.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@
 import * as crypto from "crypto";
 
 export namespace randomUtils {
-    export function getPseudononymousStringHash(s: string, encoding: crypto.HexBase64Latin1Encoding = 'base64'): string {
+    export function getPseudononymousStringHash(s: string, encoding: crypto.BinaryToTextEncoding = 'base64'): string {
```
Comment (nturinski) [inline ui/src/utils/randomUtils.ts:9]: Why was this changed?

**Review by nturinski — APPROVED** [review]

File: ui/src/utils/randomUtils.ts (hunk 1 above)
Comment (ejizba) [reply ui/src/utils/randomUtils.ts:9]: I think they just changed the name of the type between Node 12 and Node 14
